src/Networking/NexusMods.Networking.NexusWebApi/GraphQL/schema.graphql

```diff
@@ -0,0 +1,3228 @@
+schema {
+  query: Query
+  mutation: Mutation
+}
+
+"Autogenerated return type of AcceptModerationFixMutation."
```

Comment:
What is the best practice for generated `.graphql` files? Are they usually committed to source control?
This is 3k lines, and it might cause conflicts depending on how often it gets changed.

Reply:
I read the docs for the library used when looking at the PR.
Seems to be as intended. The generated files are generated manually using the CLI tool.
Unless the schema breaks on the site, or we need some new field/feature, this shouldn't need updating.

Reply:
This part of Strawberry shake is also a bit jank. The initial setup has involved me doing `dotnet clean` deleting `bin` and `obj` folders, then doing a restore on all of that, so i don't want everyone to go through that hassle. And yes, adding the schema to git does allow us to do change tracking on the schema.